routes/imageUrls.js

```diff
@@ -86,4 +145,23 @@ router.post('/getImageUrl', urlencodedParser, async(req, res) => {
     }
 })
 
+router.post('/getMoreImageUrl', urlencodedParser, async(req, res) => {
+    try {
+        const target = [req.body.target];
+        if (target.length) {
+            const [imageName] = target;
+            console.log(`- Looking for '${imageName}'.`);
+            var urls = await extracMoreUrls(imageName);
+            
+            console.log('- Retrived images:', urls.length);
+            console.log('- Done.')
+            res.send(urls);
+        }
+        res.status(400).send("An 'imageName' argument is required.");
```

Comment:
??????
when would `target.length` be zero. 

put this case into if (target.length ==0) and put the other one outside if case